variables.tf

```diff
@@ -9,6 +9,23 @@ variable "name" {
   }
 }
 
+variable "resource" {
+  description = <<-EOF
+    The identifier of the GCP Resource to monitor.
+
+    The resource can be a project, folder, or organization.
+
+    Examples: "projects/my_project-123", "folders/1234567899", "organizations/34739118321"
+  EOF
+  type        = string
+  default     = "projects/terraflood-345116" # TODO(luke): remove
```

Comment:
For your testing, having a `.auto.tfvars` file would allow you to set a default, then you could gitignore that file either in this repo or at your user level.